open-metadata-implementation/repository-services/repository-services-archive-utilities/src/main/java/org/odpi/openmetadata/repositoryservices/archiveutilities/OMRSArchiveGUIDMap.java

```diff
@@ -88,8 +88,21 @@ public String  getGUID(String id)
 
         if (guid == null)
         {
-            guid = UUID.randomUUID().toString();
-
+            boolean duplicate =true;
```

Comment:
What is it likelihood of getting a GUID clash from RandomUUID given it uses a timestamp?

Reply:
I think you are looking at an old version of the fix this code is not in the current fix